crates/error/src/vtable.rs

```diff
@@ -0,0 +1,176 @@
+use super::{ConcreteError, DynError, ErrorExt, OomOrDynErrorMut, OomOrDynErrorRef, OutOfMemory};
+use crate::boxed::try_box;
+use crate::ptr::{MutPtr, OwnedPtr, SharedPtr};
+use alloc::boxed::Box;
+use core::{any::TypeId, fmt, ptr::NonNull};
+
+/// A vtable containing the `ErrorExt` methods for some type `T`.
```

Comment:
Ah, I see this doc-comment now -- I think I had asked for an explanation of the need for a raw vtable to be added elsewhere as a comment (in review part 1) so a reference to this would be sufficient. (Thanks for the writeup!)

Reply:
Addressed in https://github.com/bytecodealliance/wasmtime/pull/12163/changes/2f327112f853cf108c1b9f5dc4cc77a15ed8c055